fares-data-build-tool/fdbt-site
# Feature/tfn 382 matching inbound outbound

# Description

Creation of outbound and inbound matching pages via the return single journey path.

As part of this pr there the matching components which have been extracted to provide usability through the three matching pages.

Some points to note:
- Currently in the matching page values are not stored on refresh if an error occurs. Example if there are 2 fare stages and we select one it should populate the selection box. But due to the way the value is stored this will require work on all matching pages so we thinking of raising a separate ticket. We need a nice way of storing values as the matching data could potentially be big so needs a bit of tech thought on this.

# Testing instructions
- Run web app and go through jounrey until you hit http://localhost:5555/fareType
- Select 'Return - Single Service' radio button and press continue
- Go through until you get to the new page http://localhost:5555/outboundMatching 
- Run error validation on here if none selected
- Fill all the fare stages required and press continue
- Should show the http://localhost:5555/inboundMatching 
- Perform error validation
- Select the fare stages press continue and should get to the thank you page

Also we need test the current matching page journey as well.

# Lighthouse testing
outbound matching page
<img width="477" alt="Screenshot 2020-05-01 at 13 56 02" src="https://user-images.githubusercontent.com/1183416/80806812-8ed5bb00-8bb3-11ea-81e3-bf2dcc2cb3bb.png">

inbound matching page
<img width="472" alt="Screenshot 2020-05-01 at 13 57 27" src="https://user-images.githubusercontent.com/1183416/80806898-ba58a580-8bb3-11ea-8491-3e6a75b325aa.png">



# Type of change


# Checklist:
